lib/Data/Chronicle/Reader.pm

```diff
@@ -43,26 +42,6 @@ to save data and another method to retrieve it. All the underlying complexities
 
 =back
 
-There are three important methods this module provides:
-
-=over 4
-
-=item C<set>
-
-Given a category, name and value stores the JSONified value in Redis and PostgreSQL database under "category::name" group and also stores current
-system time as the timestamp for the data (Which can be used for future retrieval if we want to get data as of a specific time). Note that the value
-MUST be either hash-ref or array-ref.
-
-=item C<get>
-
-Given a category and name returns the latest version of the data according to current Redis cache
-
-=item C<get_for>
-
-Given a category, name and timestamp returns version of data under "category::name" as of the given date (using a DB lookup).
-
-=back
-
 =head1 Example
```

Comment:
I know this was existing code, but could you fix the `=head1` capitalisation to be consistent? It's fairly common style that all `head1` titles be in ALL CAPITALS.